test/integration/navbar.test.js

```diff
@@ -0,0 +1,106 @@
+const SeleniumHelper = require('./selenium-helpers.js');
+
+const {
+    clickXpath,
+    findByXpath,
+    buildDriver
+} = new SeleniumHelper();
+
+let remote = process.env.SMOKE_REMOTE || false;
+let rootUrl = process.env.ROOT_URL || 'https://scratch.ly';
+
+if (remote) {
+    jest.setTimeout(60000);
+} else {
+    jest.setTimeout(10000);
+}
+
+let driver;
+
+describe('www-integration navbar links', () => {
+    beforeAll(async () => {
+        driver = await buildDriver('www-integration navbar links');
+    });
+
+    beforeEach(async () => {
+        await driver.get(rootUrl);
+    });
+
+    afterAll(async () => await driver.quit());
+
+    test('Check text of navbar items', async () => {
+        let create = await findByXpath('//li[@class="link create"]');
+        let createText = await create.getText();
+        await expect(createText).toEqual('Create');
+
+        let explore = await findByXpath('//li[@class="link explore"]');
+        let exploreText = await explore.getText();
+        await expect(exploreText).toEqual('Explore');
+
+        let ideas = await findByXpath('//li[@class="link ideas"]');
+        let ideasText = await ideas.getText();
+        await expect(ideasText).toEqual('Ideas');
+
+        let about = await findByXpath('//li[@class="link about"]');
+        let aboutText = await about.getText();
+        await expect(aboutText).toEqual('About');
+
+        let join = await findByXpath('//a[@class="registrationLink"]');
+        let joinText = await join.getText();
+        await expect(joinText).toEqual('Join Scratch');
+
+        let signIn = await findByXpath('//li[@class="link right login-item"]/a');
+        let signInText = await signIn.getText();
+        await expect(signInText).toEqual('Sign in');
+    });
+
+    test('create when signed out', async () => {
+        await clickXpath('//li[@class="link create"]');
+        let infoArea = await findByXpath('//div[@class="sprite-info_sprite-info_3EyZh box_box_2jjDp"]');
```

Comment:
This might break a lot as GUI changes. Instead, maybe we can be super specific about the XPath to the GUI component to exclude the project page view and pass on the editor mode view.

Reply:
Actually it looks like we already differentiate between editor mode and player mode GUIs by giving editor mode GUI the class `gui` while player mode gui gets a class called `guiPlayer`.  So we could just look for those classes to determine what the page is displaying.

Reply:
Okay.  I updated the test to look for the gui class for this test.  I think changing the other tests to look for the guiPlayer should happen in another pr.